user-registration-app/src/app/models/registration.ts

```diff
@@ -174,4 +174,38 @@ export class RegistrationApiResponse {
   isCurrentRegistration(): boolean {
     return this.hackathon.active;
   }
+  parseFromApiResponse(value: RegistrationApiResponse) : Registration{
```

Comment:
This should be a static method under Registration, not RegistrationApiResponse

Reply:
Also should be a public method.

Reply:
done

Reply:
Still not static